apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/components/CareerTimelineAndRecruitment.tsx

```diff
@@ -100,6 +100,7 @@ export const CareerTimelineExperience_Fragment = graphql(/* GraphQL */ `
 
 export const CareerTimelineApplication_Fragment = graphql(/* GraphQL */ `
   fragment CareerTimelineApplication on PoolCandidate {
+    ...QualifiedRecruitmentsCandidate
     id
     status
     archivedAt
```

Comment:
Oh, appears something broke

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2a0ad2d0-3462-44c7-96af-fdd6859e37fa)

Looking through queries, with all the nested fragments, got a same field defined different ways. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/19a660ed-a0cd-4a13-bd44-6b645bd7c717)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/9857a714-bc31-41c0-9ec2-b848d39e847a)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/537571d5-cc39-4f43-a010-a8271c73ad98)

Reply:
Looks like come code snuck in with main since there was no conflict :woman_facepalming: Removed in [remove unused code](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10267/commits/705a088c24a44d18860efcae54a97f7228bce092)